app/components/filter-by.js

```diff
@@ -1,29 +1,100 @@
-  import Component from '@ember/component';
-  import { inject as service } from '@ember/service';
-  import { computed } from '@ember/object';
+import Component from "@ember/component";
+import { inject as service } from "@ember/service";
+import { computed } from "@ember/object";
+import { sort } from "@ember/object/computed";
 
-  export default Component.extend({
-    store: service(),
+export default Component.extend({
+  store: service(),
 
-    allUsers: computed(function () {
-      return this.get('store').findAll('user', { reload: true });
-    }),
+  init() {
+    this._super(...arguments);
+    this.set('selectedOrder','asc'); 
+    this.sortOrder = true;
+    this.set("selectedValue", 'first_name');
 
-    myTeamMembers: computed('allUsers.isFulfilled', function () {
-      return this.allUsers.uniqBy('team');
-    }),
+  },
 
+  allUsers: computed(function () {
+    return this.get("store").findAll("user", { reload: true });
```

Comment:
Use single quotes in the component js files.

Reply:
I have fixed this change.